Pod/Classes/RZMessagingWindow.m

```diff
@@ -288,4 +289,84 @@ - (void)configurePassThroughView
 #pragma mark - RZRootMessagingViewController
 
 @implementation RZRootMessagingViewController
+
+#pragma mark - Orientation method overrides
+
+-(BOOL)shouldAutorotate
+{
+    UIViewController *topViewController = [RZRootMessagingViewController topViewController];
+    
+    if ( topViewController == self ) {
+        // If the top view controller is itself then do the same as
+        // default by returning YES.
+        return YES;
+    }
+    else {
+        return topViewController.shouldAutorotate;
+    }
+}
+
+- (NSUInteger)supportedInterfaceOrientations
+{
+    UIViewController *topViewController = [RZRootMessagingViewController topViewController];
+    
+    if ( topViewController == self ) {
+        // If the top view controller is itself then do the same as
+        // default by:
+        // 1)For iPhone, all except upside down
```

Comment:
Apple makes exceptions for things like Voice Memos (you want the microphone at the top of the device to speak into it) and the Music app (so your iPhone can be upside down in a car cup holder with the headphone jack on top).